### PR TITLE
Access okd/ocp ssh over the cluster-container

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

**Comment by qinqon** [comment]
/cc @dhiller @phoracek 

This helps with knmstate ocp lane.

File: cluster-up/container.sh [hunk 1]
```diff
@@ -2,16 +2,24 @@
...
-    POSTFIX="cluster"
-    CONTAINER=$(docker ps | grep kubevirt | grep "${KUBEVIRT_PROVIDER}-${POSTFIX}" | awk '{print $1}')
-    if [ -z $CONTAINER ]; then
-        echo "container was not found"
-        exit 0
-    fi
-    docker exec $CONTAINER bash -c "if ! grep "/root/install/auth/kubeconfig" ~/.bashrc > /dev/null; \
-                                    then echo export KUBECONFIG=/root/install/auth/kubeconfig >> ~/.bashrc; fi"
-    docker exec -it $CONTAINER bash
+    CONTAINER=${provider_prefix}-cluster
```
Comment (dhiller) [inline cluster-up/container.sh]: This looks like you moved some logic into `hack/common.sh`? I don't see the change here, so I suppose it has been moved already in another commit?
EDIT: ah, I see, you moved it into `ssh.sh`.

File: cluster-up/container.sh [hunk 2]
```diff
@@ -2,16 +2,24 @@
...
+
+    docker exec $CONTAINER bash -c "cp -f /root/install/auth/kubeconfig ~/.kube/config"
```
Comment (dhiller) [inline cluster-up/container.sh]: I suppose you need the kubeconfig at that place, but why don't you symlink?

Comment (qinqon) [reply cluster-up/container.sh]: Well proper thing would be to link stuff after cluster up somehow.

**Comment by qinqon** [comment]
@dhiller Let's add some smoke tests for all the helper scripts, but at a follow up PR.

**Comment by dhiller** [comment]
> @dhiller Let's add some smoke tests for all the helper scripts, but at a follow up PR.

SOunds good!

**Comment by dhiller** [comment]
> > @dhiller Let's add some smoke tests for all the helper scripts, but at a follow up PR.
> 
> SOunds good!

But pleasse create an issue for it so we don't forget!

File: cluster-up/container.sh (hunk 2 above)
Comment (qinqon) [reply cluster-up/container.sh]: Moved to cluster-up can you recheck ?

File: cluster-up/container.sh (hunk 1 above)
Comment (qinqon) [reply cluster-up/container.sh]: I have slim down this scripts.

**Comment by qinqon** [comment]
Let's wait for the ocp jobs

**Comment by qinqon** [comment]
/hold cancel
check-up for ocp is working fine now.
